### DISCUSSION TITLE
BytePay user guide

### DISCUSSION THREAD

**Discussion by AdaLam7** [bytepayment/bytepay#4] [General]
# Recharge

![image](https://user-images.githubusercontent.com/94216827/151191383-7694880b-bfab-4afe-afc8-c306f0b8a8be.png)

# Bind Github Repository

![image](https://user-images.githubusercontent.com/94216827/151191582-624aaea1-89be-422e-898d-acc88a8c1374.png)

![image](https://user-images.githubusercontent.com/94216827/151191829-d9b2b4f7-3a31-4d23-b6a0-564c06cbfd81.png)

After active the repository, you will see the webhook been set in your GitHub repo:

![image](https://user-images.githubusercontent.com/94216827/151192166-ecff0d3e-f0c3-4ffd-825d-94d7848ee451.png)

#  Alice create a paid task

Issue owner comments the issue: `/bytepay task 0.001DOT`, the robot will create a paid task.

![image](https://user-images.githubusercontent.com/94216827/151476724-a31a2fd4-241d-4fbf-90c2-b5ad101402a7.png)

# Bob apply the task

Using `/bytepay apply` to apply the task.

![image](https://user-images.githubusercontent.com/94216827/151476781-68760e26-ad0e-41d9-8ac3-6fb87105aaaa.png)

# Alice pay for the task

Alice pays Bob if the task is finished: `/bytepay pay`

![image](https://user-images.githubusercontent.com/94216827/151476828-a62d1a32-dec6-4a10-b418-55f7cf6d7124.png)

# Anyone transfer DOT to others

Anyone who has enough DOT in bytepay account can pay DOT to others, in an already active repo, comment any issue: `/bytepay pay Bob 10DOT`

![image](https://user-images.githubusercontent.com/94216827/151651200-d84e02bc-526b-475a-a01d-dacb47b44710.png)

# Close task

The task will be closed when the issue is closed.

# Withdraw your tokens

Set your withdraw passwd, then:

![image](https://user-images.githubusercontent.com/94216827/151514294-563ac109-6630-464b-8e22-3b1f89294113.png)

# Opensource Product NFT

![image](https://user-images.githubusercontent.com/8912557/160267761-a354e3eb-d8e2-47f1-bce9-57646a9df9f1.png)

![image](https://user-images.githubusercontent.com/8912557/160267763-d47ab645-baa8-4746-a8df-d86865a53e33.png)

![image](https://user-images.githubusercontent.com/8912557/160267778-a3e6846a-0911-48d6-8cb3-8f9f4f90fe84.png)

![image](https://user-images.githubusercontent.com/8912557/160267785-22a03af8-1541-42c1-bd4b-d99ed27887cc.png)

**Comment by cocmolla** [comment]
Mint a Opensource Product NFT
<img width="1790" alt="publish" src="https://user-images.githubusercontent.com/99471309/162461529-34f8233b-9d3c-4a9d-9b0a-eda93ef787a2.png">
Opensource Product NFT Detail
<img width="1791" alt="detail" src="https://user-images.githubusercontent.com/99471309/162460643-d8bd8351-6ff3-4a15-b50c-06d78041688c.png">
